sqlglot/parser.py

```diff
@@ -1286,6 +1286,8 @@ class Parser(metaclass=_Parser):
 
     PRIVILEGE_FOLLOW_TOKENS = {TokenType.ON, TokenType.COMMA, TokenType.L_PAREN}
 
+    DESCRIBE_STATEMENT_STYLES = {"ANALYZE", "EXTENDED", "FORMATTED", "HISTORY"}
```

Comment:
I see "HISTORY" is added for dialects.databricks

I can also remove HISTORY from here and override it in databricks parser, if necessary. And do the same for ANALYZE in mysql parser. Not sure whether this is needed 🙏

Reply:
sqlglot values brevity.

How about just "DESCRIBE_STYLES"?